### PR TITLE
Limit Call size to 200 bytes

### PR DESCRIPTION
**Pull request by ferrell-code**
The idea of this PR is to limit call length to 200 bytes for all orml pallets. It mirrors https://github.com/paritytech/substrate/pull/9418

Companion to https://github.com/AcalaNetwork/Acala/pull/1386

### CODE REVIEW

**Review by xlc — APPROVED** [review]

**Comment by ferrell-code** [comment]
I could add a integration test that checks every pallet, although there will be a test in Karura/mandala that would catch any large Calls. 

It seems that the XCM stuff has some enums that allocate (relatively) a lot memory as arguments for the extrinsics

File: authority/src/lib.rs [hunk 1]
```diff
@@ -267,7 +267,7 @@ pub mod module {
 		#[pallet::weight(T::WeightInfo::fast_track_scheduled_dispatch())]
 		pub fn fast_track_scheduled_dispatch(
 			origin: OriginFor<T>,
-			initial_origin: T::PalletsOrigin,
+			initial_origin: Box<T::PalletsOrigin>,
```
Comment (ferrell-code) [inline authority/src/lib.rs:270]: So that decreases the call size from 608 bytes to 24 bytes. Not entirely sure why but numbers don't lie

**Review by shaunxw — APPROVED** [review]
LGTM

**Comment by shaunxw** [comment]
Yeah it would be helpful to add integration tests. If there're generic typed params in dispatchable calls, call size would likely to be different between mocks and prod runtimes.

**Pull request merged**